Library/Subscription.cs

```diff
@@ -341,6 +345,29 @@ public void Postpone(DateTime nextRenewalDate)
                 ReadXml);
         }
 
+        public bool UpdateNotes(String customerNotes, String termsAndConditions, String vatReverseChargeNotes)
+        {
+
+            if (customerNotes != null)
+                CustomerNotes = customerNotes;
```

Comment:
Not sure if it's the best idea to set this on the client side subscription object. What would/should happen if the API call fails?